views/includes/page-templates/trust-details.html

```diff
@@ -3,8 +3,15 @@
 
     {% include "includes/list/errors.html" %}
 
-    <h1 class="govuk-heading-xl">{{ pageParams.title }}</h1>
-    <p class="govuk-caption-l govuk-!-margin-bottom-8">{{ pageParams.subtitle }}</p>
+    {% if isReview %}
+      <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ formData.name }}</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageTitle }}</h1>
+      <br><br>
+      <label class="govuk-label govuk-!-margin-bottom-7">{{ subtitle }}</label>
+    {% else %}
+      <h1 class="govuk-heading-xl">{{ pageParams.title }}</h1>
+      <p class="govuk-caption-l govuk-!-margin-bottom-8">{{ pageParams.subtitle }}</p>
+    {% endif %}
```

Comment:
![Screenshot 2023-10-12 at 12 06 27](https://github.com/companieshouse/overseas-entities-web/assets/117734784/0713501c-5740-4449-9237-8f152dbb2f24)

Title isn't set so I don't think the above code block works

Reply:
I've fixed this by updating util and view as well